docs/src/intro/overview.md

```diff
@@ -43,6 +43,6 @@ The *advice provider* component is responsible for supplying nondeterministic in
 The advice provider consists of three components:
 * **Advice stack** which is a one-dimensional array of field elements. Being a stack, the VM can either push new elements onto the advice stack, or pop the elements from its top.
 * **Advice map** which is a key-value map where keys are words and values are vectors of field elements. The VM can copy values from the advice map onto the advice stack as well as insert new values into the advice map (e.g., from a region of memory).
-* **Merkle sets** which contain structured data reducible to Merkle paths. Some examples of merkle sets are: Merkle tree, Sparse Merkle tree, a collection of Merkle paths. Every merkle set can be uniquely identified by its root. The VM can request Merkle paths from an merkle set, as well as update an merkle set by modifying one of its nodes (this will also change the root of the modified merkle set).
+* **Merkle store** which contain structured data reducible to Merkle paths. Some examples of Merkle sets are: Merkle tree, Sparse Merkle tree, a collection of Merkle paths. Every Merkle set can be uniquely identified by its root. The VM can request Merkle paths from an Merkle store, as well as update an Merkle set by modifying one of its nodes (this will also change the root of the modified Merkle set).
```

Comment:
I'd update this to:

**Merkle store** which contain structured data reducible to Merkle paths. Some examples of such structures are: Merkle tree, Sparse Merkle tree, a collection of Merkle paths. The VM can request Merkle paths from the Merkle store, as well as update the Merkle store by updating or merging nodes contained in the store.

Reply:
Fixed with a small modification; wdyt?

* **Merkle store** which contain structured data reducible to Merkle paths. Some examples of such structures are: Merkle tree, Sparse Merkle tree, and a collection of Merkle paths. The VM can request Merkle paths from the Merkle store, as well as mutating it by updating or merging nodes contained in the store.

Reply:
I'd change the last part of the sentence to: "... as well as mutate it by ..."

Reply:
Fixed!